src/station.cpp

```diff
@@ -261,6 +265,39 @@ void Station::MarkTilesDirty(bool cargo_change) const
 	return length;
 }
 
+/**
+ * Get the catchment size of an individual station tile.
+ * @param tile Station tile to get catchment size of.
+ * @param st Associated station of station tile.
+ * @pre IsTileType(tile, MP_STATION)
+ * @return The catchment size of the station tile.
+ */
+static uint GetTileCatchmentRadius(TileIndex tile, const Station *st)
+{
+	assert(IsTileType(tile, MP_STATION));
+
+	if (_settings_game.station.modified_catchment) {
```

Comment:
Can't we get rid of UNMODIFIED catchment and use just CA_TRAIN, CA_TRUCK and the others?
Explaining how catchment areas work would be easier.

Reply:
It's possible, but requires a savegame bump. Can easily be done separately.